Makefile

```diff
@@ -20,8 +25,12 @@ dependencies:
 	@glide --quiet update
 
 test:
-	@echo ">> running tests"
+	@echo ">> starting local redis-server on port $(REDIS_PORT)"
```

Comment:
Ok, good point! I will be checking if a redis-server is running, and if not, then I will be skipping `test` with a message. Is it ok?